### DISCUSSION TITLE
How to remove Copy/Excel/CSV/PDF/Show items and search

### DISCUSSION THREAD

**Discussion by ipapaengr** [EvotecIT/PSWriteHTML#274] [Q&A] [answered]
Hello 

I am not seeing how to disable these options, is it possible ?

**Comment by PrzemyslawKlys** [comment, marked as answer]
Just use -Buttons @() and it will remove it all. Basically with buttons you can control which button is shown and which isn't.

**Reply by ipapaengr** [reply]
Thank you !
